test/unit/factories/season.py

```diff
@@ -32,5 +32,4 @@ def teams(self, create, extracted, **kwargs):
 
         if extracted:
             # A list of teams were passed in, use them
-            for team in extracted:
-                self.teams.add(team)  # pylint: disable=no-member
+            self.teams += extracted
```

Comment:
lol this was originally copied verbatim from factory_boy's page. I'm still trying to figure out how the one-to-many relationship should work

Reply:
I got the error `Instrumented List does not have attribute add` when I passed in a list of teams. It worked once I made it `+=`.